TpmTestingPkg/TpmReplayPei/Tool/TpmReplay.py

```diff
@@ -37,8 +37,21 @@
     VALUE_FROM_EVENT,
     ALG_FROM_VALUE,
     EVENT_FROM_VALUE,
+    EV_EFI_VARIABLE_DRIVER_CONFIG,
+    EV_EFI_VARIABLE_BOOT,
+    EV_EFI_VARIABLE_BOOT2,
+    EV_EFI_VARIABLE_AUTHORITY
 )
 
+from edk2toollib.uefi.authenticated_variables_structure_support import EfiSignatureDatabase, EfiSignatureDataFactory, EfiSignatureDataEfiCertX509, EfiSignatureDataEfiCertSha256
+
+from edk2toollib.utility_functions import hexdump

```

Comment:
I don't see `hexdump` referenced anywhere. If this was only used for debug, please remove it.

```suggestion
from edk2toollib.uefi.authenticated_variables_structure_support import EfiSignatureDatabase, EfiSignatureDataFactory, EfiSignatureDataEfiCertX509, EfiSignatureDataEfiCertSha256

```